doc/classes/PoolIntArray.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		An [Array] specifically designed to hold integer values ([int]). Optimized for memory usage, does not fragment the memory.
 		[b]Note:[/b] This type is passed by value and not by reference.
+		[b]Note:[/b] This type is limited to signed 32-bit integers, which means it can only store numbers between -2,147,483,648 and 2,147,483,647. In comparison, [int] uses signed 64-bit integers which can hold much larger values.
```

Comment:
I'd mention that it's `[-2^31, 2^31 - 1]` and write the numbers without thousand separators, which can be confusing depending on your locale.
See how it's done in `int`, we should stay consistent:
```
classes/int.xml
8:              It can take values in the interval [code][-2^63, 2^63 - 1][/code], i.e. [code][-9223372036854775808, 9223372036854775807][/code]. Exceeding those bounds will wrap around.
```